lib/android-helpers.js

```diff
@@ -447,14 +447,15 @@ helpers.unlockWithUIAutomation = async function (driver, adb, unlockCapabilities
     throw new Error(`Invalid unlock type ${unlockType}`);
   }
   let unlockKey = unlockCapabilities.unlockKey;
-  if (!unlocker.isValidKey(unlockType, unlockKey)) {
+  if (!unlocker.isValidKey(unlockType, unlockKey) && unlockType !== SWIPE_UNLOCK) {
```

Comment:
why SWIPE_UNLOCK cannot be added to `isValidKey` verification?

Reply:
we dont need a to validate a key for swipe unlock, since there's no need to set this param. I'll change this to be more comprehensible.